### PR TITLE
Bug: LT-2040 manage organisation (user) page performance

### PR DESCRIPTION
**Pull request by Mark-Spiers-kainos**
Minimise the amount of data being passed forward to the page, increasing the speed greatly. Deployed on QA.

### CODE REVIEW

File: gov_users/serializers.py [hunk 1]
```diff
@@ -24,6 +24,12 @@ class Meta:
         )
 
 
+class RoleNameSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Role
+        fields = ("id", "name")
```
Comment (andytaylorkainos) [inline gov_users/serializers.py]: 👀 comma at the end of the tuple please

File: organisations/serializers.py [hunk 2]
```diff
@@ -213,3 +214,17 @@ class ExternalLocationSerializer(serializers.ModelSerializer):
...
         fields = ("id", "name", "address", "country", "organisation")
+
+
+class OrganisationUserListView(serializers.ModelSerializer):
+    status = serializers.CharField()
+    role = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ExporterUser
+        fields = ("id", "first_name", "last_name", "email", "role", "status")
```
Comment (andytaylorkainos) [inline organisations/serializers.py]: comma

File: gov_users/serializers.py (hunk 1 above)
Comment (jfaracik) [reply gov_users/serializers.py]: Is there anyway we could get Black to do this? Purpose of Black was to avoid conversations like this

**Comment by jfaracik** [comment]
> Minimise the amount of data being passed forward to the page, increasing the speed greatly. Deployed on QA.

How much faster will the user's page be after this change?

**Pull request merged**